packages/py3_identify.rb

```diff
@@ -0,0 +1,14 @@
+require 'buildsystems/pip'
+
+class Py3_identify < Pip
+  description 'File identification library for Python'
+  homepage 'https://github.com/pre-commit/identify'
+  version '2.6.0-py3.12'
+  license 'MIT'
+  compatibility 'all'
+  source_url 'SKIP'
+
+  depends_on 'python3' => :build
+
+  no_compile_needed
```

Comment:
I'm thinking why not add `depends_on 'python3' => :build` and `no_compile_needed` to the pip buildsystem since all python packages installed via pip need these anyway?  It would reduce redundancy.

Reply:
Also, most of them probably depend on python for more than building.